requirements.txt

```diff
@@ -1,4 +1,4 @@
 sqlite3@3.17 -DCMAKE_POSITION_INDEPENDENT_CODE=On
 boost@1.72 -DCMAKE_POSITION_INDEPENDENT_CODE=On --build
 half,https://github.com/pfultz2/half/archive/1.12.0.tar.gz -X header -H sha256:0a08660b68abb176ebc2a0cdf8de46e3182a7f46c66443bb80dbfaaec98cf969 --build
-ROCmSoftwarePlatform/llvm-project-mlir@release/rocm-5.1 -DBUILD_FAT_LIBMLIRMIOPEN=1
+ROCmSoftwarePlatform/llvm-project-mlir@fp16-wrw-optimization-rebase -DBUILD_FAT_LIBMLIRMIOPEN=1
```

Comment:
Will do.

Reply:
I might not make it clear. I don't think we need that comment there. Instead, since your wrw fp16 PR in MLIR has already been merged to the mainline, you can just pick up the latest commit in our HEAD and make sure this feature is included, such that:

> ROCmSoftwarePlatform/llvm-project-mlir@b04c00925237718e5d260efb2a1776ee6f99f5b4 -DBUILD_FAT_LIBMLIRMIOPEN=1

Reply:
Pushed another commit to address this.